WebApp/load.py

```diff
@@ -0,0 +1,7 @@
+import google.generativeai as genai
+def output(text):
+    response = model.generate_content("Summarize this and also don't make this too short "+text)
+    return response
+GOOGLE_API_KEY="AIzaSyCgX2dUhfUNGZ1aSrjUIlXNfor1ylBdw7I"

```

Comment:
Move API keys, credentials to .env file. This is a bad coding practice, sensitive data should not be made public.